cli/tools/coverage/mod.rs

```diff
@@ -602,7 +604,8 @@ fn filter_coverages(
         || e.url.starts_with(npm_root_dir)
         || e.url.ends_with("__anonymous__")
         || e.url.ends_with("$deno$test.js")
-        || e.url.ends_with(".snap");
+        || e.url.ends_with(".snap")
+        || is_supported_test_ext(Path::new(e.url.as_str()));
```

Comment:
Prefer to use `if Ok(path) = e.url.to_file_path()` here

Reply:
`e.url` is a String - so this won't work?